docs/source-app/workflows/run_app_on_cloud/cloud_files.rst

```diff
@@ -39,6 +38,18 @@ For example, the source code directory below with the ``.lightningignore`` file
 
 A sample ``.lightningignore`` file can be found `here <https://github.com/Lightning-AI/lightning.beta/blob/master/.lightningignore>`_.
 
+If you are a component author and your components creates local files that you want to ignore, you can do:
```

Comment:
Can you revert #15818? Breaks local apps that set the argument until https://github.com/Lightning-AI/lightning/pull/16057 lands.

Reply:
Nvm. #16057 landed. I cherry-picked it

Reply:
@Borda please revert #15818 too